back/src/database/SpotSQL.js

```diff
@@ -1,43 +1,64 @@
 'use strict';
+const con = require('./DBHandler.js');
+const connection = con.connection;
+const {info, debug, warning, error}  = require('../winston');
+// const { connect } = require('../route/userRoute.js');
+const fileLabel = "SpotSQL"
+const Spot = require('../objects/spot');
+module.exports = {saveSpot:saveSpot};
```

Comment:
Does this work even if the method is declared after the exports?
Usually we put the exports at the end of the file